### PR TITLE
[Import/Export CSV] Use rewrites for accessing HTTPS functions

### PR DESCRIPTION
**Pull request by gino-m**

### CODE REVIEW

**Comment by gino-m** [comment]
@DakshSaini PTAL?

File: web-ng/src/app/services/data-import/data-import.service.ts [hunk 1]
```diff
@@ -35,11 +35,13 @@ export class DataImportService {
     projectId: string,
     layerId: string,
     file: File
-  ): Observable<ImportCsvResponse> {
+  ): Promise<ImportCsvResponse> {
```
Comment (DakshSaini) [inline web-ng/src/app/services/data-import/data-import.service.ts:38]: We should remove unused `Observable` import statement from line 20?

**Review by DakshSaini — APPROVED** [review]
LGTM! :shipit: 🚢

File: web-ng/src/app/services/data-import/data-import.service.ts (hunk 1 above)
Comment (gino-m) [reply web-ng/src/app/services/data-import/data-import.service.ts:38]: Good catch!

**Comment by gino-m** [comment]
> LGTM! :shipit: 🚢

RFAL!

**Review by DakshSaini — APPROVED** [review]
Ready to 🚢 :)

**Comment by DakshSaini** [comment]
> > LGTM! :shipit: 🚢
> 
> RFAL!

Sorry, I'm afraid I'm not sure what RFAL means, 😅

**Pull request merged**